src/servers/ZoneServer2016/managers/constructionmanager.ts

```diff
@@ -453,26 +452,26 @@ export class ConstructionManager {
   ): string {
     // for construction entities that don't have a parentObjectCharacterId from the client
     let freeplaceParentCharacterId = "";
+    // TODO: SEARCH FOUNDATIONS IN GRID RANGE INSTEAD OF ALL OF THEM
```

Comment:
SEARCH FOUNDATIONS IN GRID RANGE INSTEAD OF ALL OF THEM
 ( so i can auto-create an issue for it )